pkg/oc/lib/newapp/app/imageref.go

```diff
@@ -347,7 +336,6 @@ func (r *ImageRef) ImageStreamTag() (*imageapi.ImageStreamTag, error) {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:        istname,
 			Annotations: map[string]string{"openshift.io/imported-from": r.Reference.Exact()},
-			Namespace:   r.SuggestNamespace(),
```

Comment:
why is this no longer needed? it seems like a useful disambiguation.

Reply:
setting namespace in another function to c.OriginNamespace if one doesn't exist